keys/api_kerberos.go

```diff
@@ -70,20 +70,32 @@ type KerberosAPI interface {
 	KerberosReadExecute(r ApiKerberosReadRequest) (*KeysReadKerberosKeyResponse, *http.Response, error)
 
 	/*
-			KerberosUpdate Update the Kerberos key.
+		KerberosUpdate Update the Kerberos key.
 
-			Use this method to update a __KerberosKey__ object.
-		A __KerberosKey__ object represents a Kerberos key.
+		Use this method to update a __KerberosKey__ object.
+	A __KerberosKey__ object represents a Kerberos key.
 
-			@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
-			@param id An application specific resource identity of a resource
-			@return ApiKerberosUpdateRequest
+		@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+		@param id An application specific resource identity of a resource
+		@return ApiKerberosUpdateRequest
 	*/
 	KerberosUpdate(ctx context.Context, id string) ApiKerberosUpdateRequest
 
 	// KerberosUpdateExecute executes the request
 	//  @return KeysUpdateKerberosKeyResponse
 	KerberosUpdateExecute(r ApiKerberosUpdateRequest) (*KeysUpdateKerberosKeyResponse, *http.Response, error)
+
+	/*
+		KeysKerberosPost Method for KeysKerberosPost
+
+		@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+		@return ApiKeysKerberosPostRequest
+	*/
+	KeysKerberosPost(ctx context.Context) ApiKeysKerberosPostRequest
```

Comment:
This change is unexpected for this PR. 

Maybe Post method was excluded from the schema, the first time it was generated, and the changes were not pushed.